src/redux/reducers/samples/initialState.js

```diff
@@ -0,0 +1,26 @@
+const sampleTemplate = {
+  name: null,
+  uuid: null,
+  type: null,
+  species: null,
+  complete: false,
+  error: false,
+  fileNames: [],
+  files: {},
```

Comment:
can you add a `createdDate` and `lastModified` field? These fields should store `moment().isoTime()`